baybe/kernels/base.py

```diff
@@ -17,12 +19,20 @@
 class Kernel(ABC, SerialMixin):
     """Abstract base class for all kernels."""
 
+    lengthscale_prior: Optional[Prior] = field(default=None, kw_only=True)
+    """An optional prior that can be applied to the lengthscale."""
```

Comment:
I think "apply" sounds a bit weird here
```suggestion
    """An optional prior on the kernel lengthscale."""
```

Reply:
I think I took that from the gpytorch formulation, will adjust :)